OpenRA.Mods.Common/Activities/Air/Fly.cs

```diff
@@ -50,6 +56,12 @@ public Fly(Actor self, Target t, WPos? initialTargetPosition = null, Color? targ
 				lastVisibleTarget = Target.FromPos(target.CenterPosition);
 			else if (initialTargetPosition.HasValue)
 				lastVisibleTarget = Target.FromPos(initialTargetPosition.Value);
+
+			lastFacing = self.TraitOrDefault<IFacing>().Facing;
```

Comment:
`TraitOrDefault` means that this can be null so retrieving `Facing` will crash.

```suggestion
			var facing = self.TraitOrDefault<IFacing>()
			if (facing != null)
				lastFacing = facing.Facing;
```

or

```suggestion
			lastFacing = self.Trait<IFacing>().Facing;
```

Reply:
`Aircraft` implements `IFacing` so this is guaranteed to not be null - please use @Mailaender's second suggestion.